addons/storyshots/stories/directly_required/index.js

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { storiesOf } from '@storybook/react'; // eslint-disable-line
 import { action } from '@storybook/addon-actions'; // eslint-disable-line
 
-import Button from '@storybook/components/dist/demo/Button';
+import Button from '@storybook/react/dist/demo/Button';
```

Comment:
investigated that test fails after this change
not sure what the reason, possible someone who more familiar with storyshots could help?